readme/headless-clipper.md

```diff
@@ -0,0 +1,19 @@
+HeadlessClient contains a headless clipper server.
+
+This service can help joplin mobile app to convert URLs shared from other apps into clipped simplied page.
+
+When this headless client sync from various sync target periosely, it watches created and updated notes with only URL in body, and converts such notes into clipped simplied page, just like what joplin-clipper + ClipperServer is doing on browser webExtention and ElectronClient.
```

Comment:
periosely ? is there a typo ?

Reply:
I think 
```suggestion
periodically
```

was meant. In any case, the entire paragraph could need a rewrite, but I understood what the author meant to say, so these changes are just cosmetically.